sentry/src/main/java/io/sentry/Breadcrumb.java

```diff
@@ -663,8 +667,11 @@ public void setUnknown(@Nullable Map<String, Object> unknown) {
   @Override
   @SuppressWarnings("JavaUtilDate")
   public int compareTo(@NotNull Breadcrumb o) {
-    // TODO also use nano time if equal
-    return timestamp.compareTo(o.timestamp);
+    int timestampCompare = timestamp.compareTo(o.timestamp);
```

Comment:
In order to have consistent ordering, should we always use nanos?
We would also have to change the constructor to take a nano time.

Reply:
Yeah makes sense, e.g. for OTEL where we likely have to create breadcrumbs after the fact. Will change it to only use `nanos` in a follow up PR.